modal/cli/volume.py

```diff
@@ -89,8 +90,7 @@ async def ls(volume_name: str, path: str = typer.Argument(default="/")):
         entries = await volume.listdir(path)
     except GRPCError as exc:
         if exc.status in (Status.INVALID_ARGUMENT, Status.NOT_FOUND):
-            print(exc.message, file=sys.stderr)
-            return
+            raise UsageError(exc.message)
```

Comment:
@freider seems we could use `UsageError` in a few places where we `print` and then `exit`. Is there any reason we shouldn't?